### ISSUE TITLE
Bump fourseven:scss to remove underscore warning

### ISSUE THREAD

**Issue by janzheng** [themeteorchef/bert#51]
I'm trying out one of your examples and accidentally updated meteor to 1.5... and subsequently probably updated ```fourseven:scss``` and now I'm getting the problem:

``` While processing files with fourseven:scss (for target web.browser):
   /packages/themeteorchef_bert/stylesheets/bert.scss: Scss compiler error: _.without is not a
   function```

**Comment by RickvdP** [comment]
+1

**Comment by themeteorchef** [comment]
Ran into this, too, and found the easiest fix is to pin the `fourseven:scss` package to its `4.5.0` release in `./meteor/versions` (in your app directly) like this: `fourseven:scss@4.5.0`. My guess is the global underscore `_` variable went missing in Meteor 1.5.

**Comment by ChiragWeb** [comment]
Hi Chef, I was also facing the problem. I tried this option and its resolved ..Thanks for the quick fix.

**Comment by hwillson** [comment]
Quick FYI - `fourseven:scss` has fixed this issue in version 4.5.4 (see https://github.com/fourseven/meteor-scss/issues/252).

**Comment by themeteorchef** [comment]
Thanks @hwillson :)

**Comment by themeteorchef** [comment]
Fixed in `v2.1.3`.
